browser/mapping.go

```diff
@@ -219,11 +219,18 @@ func mapElementHandle(vu moduleVU, eh *common.ElementHandle) mapping {
 	maps := mapping{
 		"boundingBox": eh.BoundingBox,
 		"check":       eh.Check,
-		"click": func(opts goja.Value) *goja.Promise {
+		"click": func(opts goja.Value) (*goja.Promise, error) {
+			ctx := vu.Context()
+
+			popts := common.NewElementHandleClickOptions(eh.Timeout())
```

Comment:
nit: What does `popts` stand for? Should it be `clickOpts`?

Reply:
I think `copts` would be fine for "click options".

Reply:
`popts` -> `parsed options`. Aren't `opts` before being parsed also click options? I prefer `popts`.